src/main/java/com/coreos/jetcd/EtcdClientBuilder.java

```diff
@@ -54,14 +59,48 @@ public EtcdClientBuilder endpoints(String ... endpoints) {
         return this;
     }
 
+    public ByteString getName() {
+        return name;
+    }
+
+    /**
+     * config etcd auth name
+     *
+     * @param name etcd auth name
+     * @return this builder
+     * @throws NullPointerException if name is null
+     */
+    public EtcdClientBuilder setName(ByteString name) {
```

Comment:
@hongchaodeng , We use this as this is the string used by gRPC which support many encoding.